packages/next/src/next-devtools/dev-overlay/components/errors/call-stack/call-stack.tsx

```diff
@@ -1,13 +1,16 @@
 import type { OriginalStackFrame } from '../../../../shared/stack-frame'
 import { useMemo, useState, useRef } from 'react'
-import { CallStackFrame } from '../../call-stack-frame/call-stack-frame'
+import { CallStack } from '../../call-stack/call-stack'
 
 interface CallStackProps {
   frames: OriginalStackFrame[]
   dialogResizerRef: React.RefObject<HTMLDivElement | null>
 }
 
-export function CallStack({ frames, dialogResizerRef }: CallStackProps) {
+export function ErrorOverlayCallStack({
```

Comment:
Should probably rename this file to a more special name that align with the export name, as it's quite confusing with another headless callstack/callstack component